assets/less/hel/hearing-page.less

```diff
@@ -3,6 +3,8 @@
     padding: 15px;
     font-size: 1.4rem;
     margin-bottom: 25px;
+    overflow-y: scroll;
```

Comment:
Let's lock the sidebar scrolling to the main text. At the moment, nobody realizes the sidebar is scrollable, so the rest of the sidebar will remain unseen by users.